1.data-exploration/scripts/nbconverted/test-train_split.py

```diff
@@ -0,0 +1,180 @@
+
+
+# In[6]:
+
+
+ref_df.loc[ref_df["age_categories"] == "Pediatric"].reset_index(drop=True)
+
+
+# In[7]:
+
+
+# data frame containing ALL the PEDIATRIC samples
+bulk_pediatric_training_df = ref_df.loc[ref_df["age_categories"] == "Pediatric"].reset_index(drop=True)
+print(bulk_pediatric_training_df.shape)
+bulk_pediatric_training_df.head(3)
+
+
+# In[8]:
+
+
+# data frame containing ALL the ADULT samples
+bulk_adult_training_df = ref_df.loc[ref_df["age_categories"] == "Adult"].reset_index(drop=True)
+print(bulk_adult_training_df.shape)
+bulk_adult_training_df.head(3)
+
+
+# In[9]:
+
+
+# sorting out 103 rows (85% of the PEDIATRIC samples) for the TRAINING data frame
+
+pre_merge_pediatric_training_df = bulk_pediatric_training_df[0:103].reset_index(drop=True)
+print(pre_merge_pediatric_training_df.shape)
+pre_merge_pediatric_training_df.head(3)
+
+
+# In[10]:
+
+
+# sorting out 18 rows (15% of the PEDIATRIC samples) for the TESTING data frame
+pre_merge_pediatric_testing_df = bulk_pediatric_training_df[103:].reset_index(drop=True)
+print(pre_merge_pediatric_testing_df.shape)
+pre_merge_pediatric_testing_df.head(3)
+
+
+# In[11]:
+
+
+# sorting out 649 rows (85% of the ADULT samples) for the TRAINING data frame
+pre_merge_adult_training_df = bulk_adult_training_df[0:649]
+print(pre_merge_adult_training_df.shape)
+pre_merge_adult_training_df.head(3)
+
+
+# In[12]:
+
+
+# sorting out 114 rows (15% of the ADULT samples) for the TESTING data frame
+pre_merge_adult_testing_df = bulk_adult_training_df[649:].reset_index(drop=True)
+print(pre_merge_adult_testing_df.shape)
+pre_merge_adult_testing_df.head(3)
+
+
+# In[13]:
+
+
+# merging the TRAINING data frames 
+training_merge_frames = [pre_merge_adult_training_df, pre_merge_pediatric_training_df]
+training_df_IDs = pd.concat(training_merge_frames).reset_index(drop=True)
+training_df_IDs
+
+
+# In[14]:
+
+
+# merging the TRAINING data frames 
+testing_merge_frames = [pre_merge_adult_testing_df, pre_merge_pediatric_testing_df]
+testing_df_IDs = pd.concat(testing_merge_frames).reset_index(drop=True)
+testing_df_IDs
+
+
+# In[15]:
+
+
+print(dependency_df.shape)
+dependency_df.head()
+
+
+# In[16]:
+
+
+# searching for similar IDs FROM the training_df_IDs IN the dependency_df
+training_df_IDs = training_df_IDs["DepMap_ID"].tolist()
+training_df_IDs = set(training_df_IDs) & set(dependency_df["DepMap_ID"].tolist())
+
+
+# In[17]:
+
+
+training_df = dependency_df.loc[dependency_df["DepMap_ID"].isin(training_df_IDs)].reset_index(drop=True)
+print(training_df.shape)
+training_df.head(3)
+
+
+# In[18]:
+
+
+# searching for similar IDs FROM the testing_df_IDs IN the dependency_df
+testing_df_IDs = testing_df_IDs["DepMap_ID"].tolist()
+testing_df_IDs = set(testing_df_IDs) & set(dependency_df["DepMap_ID"].tolist())
+
+
+# In[19]:
+
+
+testing_df = dependency_df.loc[dependency_df["DepMap_ID"].isin(testing_df_IDs)].reset_index(drop=True)
+print(testing_df.shape)
+testing_df
+
```

Comment:
you need to somehow save this info - probably in a new column in the sample_info file